docs/static/monitoring-apis.asciidoc

```diff
@@ -0,0 +1,312 @@
+
+//QUESTION: Do you think we should describe the fields that are returned by each API? I haven't done this because I feel that the fields are intuitively named and that the descriptions would be redundant. WDYT?
+
+You can use the `?human` parameter to return the document in a human-readable format.
+
+[source,js]
+--------------------------------------------------
+GET /_node/hot_threads?human=true
+--------------------------------------------------
+//AUTOSENSE
+
+Example of a human-readable response: 
+
+[source,js]
+--------------------------------------------------
+::: {Ringo Kid}{Gv3UrzR3SqmPQIgfG4qJMA}{127.0.0.1}{127.0.0.1:9300}
+   Hot threads at 2016-01-13T16:55:49.988Z, interval=500ms, busiestThreads=3, ignoreIdleThreads=true:
+
+    0.0% (216micros out of 500ms) cpu usage by thread 'elasticsearch[Ringo Kid][transport_client_timer][T#1]{Hashed wheel timer #1}'
+     10/10 snapshots sharing following 5 elements
+       java.lang.Thread.sleep(Native Method)
+       org.jboss.netty.util.HashedWheelTimer$Worker.waitForNextTick(HashedWheelTimer.java:445)
+       org.jboss.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:364)
+       org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
+       java.lang.Thread.run(Thread.java:745)
+
+    0.0% (216micros out of 500ms) cpu usage by thread 'elasticsearch[Ringo Kid][transport_client_timer][T#1]{Hashed wheel timer #1}'
+     10/10 snapshots sharing following 5 elements
+       java.lang.Thread.sleep(Native Method)
+       org.jboss.netty.util.HashedWheelTimer$Worker.waitForNextTick(HashedWheelTimer.java:445)
+       org.jboss.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:364)
+       org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
+       java.lang.Thread.run(Thread.java:745)
+--------------------------------------------------
+
+/////
+Commenting this section out because it looks like this API was moved to Alpha 2. Feel free to add review
+comments, tho, if you notice inaccuracies.
+
+[[plugins-api]]
+=== Plugins API
+
+experimental[]
+
+The plugins API gets information about all Logstash plugins that are currently installed.
+This API basically returns the output of running the `bin/plugins list --verbose` command.
+
+[source,js]
+--------------------------------------------------
+GET /_plugins
+--------------------------------------------------
+//AUTOSENSE
+
+The output is a JSON document.
+
+Example response:
+
+[source,js]
+--------------------------------------------------
+"total": 102
+"plugins" : [
+  {
+      "name": "logstash-output-pagerduty"
+      "version": "2.0.2"
+  },
+  {
+      "name": "logstash-output-elasticsearch"
+      "version": "2.1.2"
+  }
+....
+] 
+--------------------------------------------------
+/////
+
+[[root-resource-api]]
+=== Root Resource API
+
+experimental[]
+
+//QUESTION: Is "root resource API" a good name for this API? 
```

Comment:
I think this is an OK name, I wish naming thing was easier from time to time ;)
@purbon do you have a better name for this one?